pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -133,3 +136,18 @@ func getNetworkClass(network *v1.Network) (NetworkInterface, error) {
 	}
 	return nil, fmt.Errorf("Network not implemented")
 }
+
+// IsIpv6Enabled returns if IPv6 is enabled
+func IsIpv6Enabled() bool {
+	podIP := os.Getenv("MY_POD_IP")
```

Comment:
This is used by both the virt-handler and the virt-launcher so the question is they both have this environment variable?

Reply:
I'm also confused. What sets this variable?

Reply:
please view the comment in the other thread which also refer to this question